src/gorilla.c

```diff
@@ -115,17 +115,19 @@
 
 // 2^bit
 static inline u_int64_t BIT(u_int64_t bit) {
-    return 1ULL << bit;
+    __uint128_t tmp = 1;
```

Comment:
@ashtul can we add unit tests to these functions?

Reply:
@danni-m 
We defiantly should.
We need to discuss the platform for all unit tests.